server/graphql/loaders.js

```diff
@@ -2,15 +2,11 @@ import models, { sequelize, Op } from '../models';
 import { getListOfAccessibleMembers } from '../lib/auth';
 import { TransactionTypes } from '../constants/transactions';
 import DataLoader from 'dataloader';
-import dataloaderSequelize from 'dataloader-sequelize';
+import { createContext } from 'dataloader-sequelize';
+
 import { get, groupBy } from 'lodash';
 import debugLib from 'debug';
 
-dataloaderSequelize(models.Order);
-dataloaderSequelize(models.Transaction);
-dataloaderSequelize(models.Collective);
-dataloaderSequelize(models.Expense);
```

Comment:
My understanding is that this was enabling automatic dataloading within Sequelize. For example, all calls to `models.Collective.findByPk` would be cached for the request thanks to that.

If that's right, it means that all call to `models.Collective.findByPk` in the GraphQL API needs to be replaced by `req.loaders.Collective.byId.load`.

Reply:
Hi, thanks for pointing this out, had to revisit how dataloaders work. My previous understanding was that they were used further down during the request when data had already been called & cached from sequelize, not automatically. I've made some changes for the Collective loaders.